charts/incubator/linkwarden/values.yaml

```diff
@@ -0,0 +1,70 @@
+image:
+  repository: ghcr.io/linkwarden/linkwarden
+  pullPolicy: IfNotPresent
+  tag: v2.2.1@sha256:c948cdcf683ec7b6c634eb2a0e184e270e167401d41ef8e82f7ff64c61f39f9c
+
+config:
+  pagination_take_count: 20
+  autoscroll_timeout: 30
+  disable_registration: false
+  archive_limit: 5
+  smtp:
+    enabled: false
+    from: ""
+    server: ""
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+    runAsUser: 0
+
+service:
+  main:
+    ports:
+      main:
+        port: 2985
+        targetPort: 3000
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            DATABASE_URL:
+              secretKeyRef:
+                name: cnpg-main-urls
+                key: std
+            POSTGRES_PASSWORD:
+              secretKeyRef:
+                name: cnpg-main-user
+                key: password
+            NEXTAUTH_SECRET:
+              secretKeyRef:
+                name: linkwarden-secrets
+                key: NEXTAUTH_SECRET
+            NEXTAUTH_URL: "http://localhost:3000/api/v1/auth"
+            PAGINATION_TAKE_COUNT: "{{ .Values.config.pagination_take_count }}"
+            STORAGE_FOLDER: "/data"
+            AUTOSCROLL_TIMEOUT: "{{ .Values.config.autoscroll_timeout }}"
+            NEXT_PUBLIC_DISABLE_REGISTRATION: "{{ .Values.config.disable_registration }}"
+            RE_ARCHIVE_LIMIT: "{{ .Values.config.archive_limit }}"
+            NEXT_PUBLIC_EMAIL_PROVIDER: "{{ .Values.config.smtp.enabled }}"
+            EMAIL_FROM: "{{ .Values.config.smtp.from }}"
+            EMAIL_SERVER: "{{ .Values.config.smtp.server }}"
+
+persistence:
+  data:
+    enabled: true
+    mountPath: "/data/data"
```

Comment:
Should this match to STORAGE_FOLDER?

Reply:
I did that in comparison to their docker-compose.yml which had /data/data for the volume https://github.com/linkwarden/linkwarden/blob/main/docker-compose.yml
And the STORAGE_FOLDER had just /data
I could be understanding this wrong

Reply:
from what i read, i think just creating a aditonal storage mount for example: "/storage" and make the variable default to that mountpath instead.

Reply:
When I try setting the persistence to just /data, it ends up throwing errors

```
2023-11-17 16:19:31.812581-08:00yarn run v1.22.19
2023-11-17 16:19:31.830248-08:00error Couldn't find a package.json file in "/data"
```

Reply:
https://github.com/linkwarden/linkwarden/blob/b8683185481dd03a5dd35df40d34f79a70ef97d8/lib/api/storage/createFolder.ts#L9C1-L10

Remove the `/` from `STORAGE_FOLDER`
It just appends it to the _WORKDIR_ which is `/data`
https://github.com/linkwarden/linkwarden/blob/b8683185481dd03a5dd35df40d34f79a70ef97d8/Dockerfile#L7

Reply:
The mountPath for the volume still has to be /data/data or else the error returns.

Reply:
Yes thats correct